po/vi.po

```diff
@@ -139,23 +139,23 @@ msgstr "Labels (nhãn)"
 
 #: src/SUMMARY.md src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks and Scopes"
-msgstr ""
+msgstr "Blocks (khối lệnh) và scopes (phạm vi)"
```

Comment:
We have a rough translation of `block` and `scope` in the `Glossary`, so we can use those terms as is in my opinion.

Reply:
Ok, removed the Vietnamese translations